packages/js/src/elementor/initializers/editor-store.js

```diff
@@ -17,8 +17,8 @@ const populateStore = store => {
 	store.dispatch( actions.loadCornerstoneContent() );
 	// Initialize the focus keyphrase.
 	store.dispatch( actions.loadFocusKeyword() );
-	// Hide marker buttons.
-	store.dispatch( actions.setMarkerStatus( "hidden" ) );
+	// Enable marker buttons.
+	store.dispatch( actions.setMarkerStatus( "enabled" ) );
```

Comment:
OK to keep this shortcut (should ideally have lived in Premium if we wouldn't have had an upsell). For reference, it's also how it's implemented in https://github.com/Yoast/wordpress-seo/pull/20846.